client/js/components/fragments/Map.js

```diff
@@ -31,11 +31,10 @@ class Map extends Component {
   }
 
   configureMap(google) {
-    this.map = new google.maps.Map(document.getElementById('map'), {
-      center: {lat: 0, lng: 50},
-      zoom: 2,
-      minZoom: 2,
-      mapTypeId: google.maps.MapTypeId.SATELLITE,
+    this.map = new google.maps.Map(document.getElementById('Map'), {
+      center: {lat: 0, lng: 0},
+      zoom: 3,
```

Comment:
I set zoom to 2 as then I could see the whole world by default on my laptop :) What screen are you using? With zoom=3 I can't see the whole world by default. Maybe we could somehow decide the default zoom by checking user's resolution or sth?